app/webpacker/packs/application.js

```diff
@@ -9,6 +9,14 @@ new CookieMessage($cookieMessage).init();
 
 var $copyWarningMessage = document.querySelector('[data-copy-course="warning"]');
 if ($copyWarningMessage) {
+  var $enrichmentForm = document.querySelector('[data-qa="enrichment-form"]');
```

Comment:
`data-qa` is for testing hooks, this is for hooking in a behaviour. We could use an `id` instead, but it's not a big deal.